requirements.txt

```diff
@@ -12,3 +12,5 @@ pytest==3.9.3
 pytest-benchmark>=3.1
 pytest-cov>=2.6.0
 coveralls>=1.5.1
+seaborn>=0.9.0
```

Comment:
this should not be in base library requirements

Reply:
Yep worked. I uninstalled seaborn, removed seaborn from requirements.txt and added seaborn to setup.py demos, and installing with
pip install stellargraph[demos]
then tested:
-> ensemble-node-classification-example.ipynb worked with seaborn 

I think this change just needs the README changed to tell people who install with git that they still have to run:
pip install stellargraph[demos]

Reply:
@youph I've made the changes. and pushed it up.